src/patternfly/upgrade-examples/AlertUpgradeExamples/docs/code.md

```diff
@@ -0,0 +1,18 @@
+## Overview
+
+When converting PatternFly 3 components to PatternFly 4 components, you must also take into consideration the layouts and sizings that PatternFly 3 utilized from Bootstrap 3.
+
+## Usage
+
+| PF3 Class | Applied To | PF4 Class | Applied To | Outcome |
+| -- | -- | -- | -- | -- |
+| `.alert` | `<div>` | `.pf-c-alert` | `<div>` | Applies default alert styling. Always use with a modifier class. Required. |
+| `.alert-success` | `<div>` | `.pf-m-success` | `.pf-c-alert` | Applies success styling.|
+| `.alert-danger` | `<div>` | `.pf-m-danger` | `.pf-c-alert` | Applies danger styling. |
+| `.alert-warning` | `<div>` | `.pf-m-warning` | `.pf-c-alert` | Applies warning styling. |
+| `.alert-info` | `<div>` | `.pf-m-info` | `.pf-c-alert` | Applies info styling.|
+| `.alert-link` | `<a>` | `.pf-c-alert-action` | `.pf-c-alert` | Applies link styling. |
```

Comment:
I think you can remove the PF4 class/applied to since you're just using a normal `<a>` there now for inline links in the alert body text.

Reply:
removed PF4 class/applied